tracker/templates/advisory.html

```diff
@@ -17,18 +17,48 @@ <h1>{{ advisory.id }}
 				{%- endif %}
 				<a href="/{{  advisory.id }}/{% if generated %}generate/{% endif %}raw" accesskey="r">raw</a>
 			</h1>
-			<table class="styled-table full size">
-				<thead>
-					<tr>
-						<th>[{{ advisory.id }}] {{package.pkgname}}: {{advisory.advisory_type}}</th>
-					</tr>
-				</thead>
-				<tbody>
-					<tr>
-						<td>
-							<div class='advisory'>{{ raw_asa|safe|urlize }}</div>
-						</td>
-					</tr>
-				</tbody>
-			</table>
+			<div class="row">
+				<div class="one-half column">
+					<table class="styled-table full size">
+						<thead>
+							<tr>
+								<th>[{{ advisory.id }}] {{package.pkgname}}: {{advisory.advisory_type}}</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td>
+									<div class='advisory'>{{ raw_asa|safe|urlize }}</div>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+				</div>
+
+				{%- if current_user.is_authenticated and form %}
```

Comment:
what's the intention of this `and form`?

Reply:
`render_html_advisory` is a function that is called two placed. Once for `/generate` and another once published. Since we only want the form under `/generate` it's a nice short circuit.